### ISSUE TITLE
br.com.vidaas.sp - broken app

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardForAndroid#4026]
@adguard-bot commented on [Mon Nov 01 2021](https://github.com/AdguardTeam/AdguardFilters/issues/98667)

### Issue URL (Incorrect Blocking)
 [https://play.google.com/store/apps/details?id=br.com.vidaas.sp](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dbr.com.vidaas.sp)
### Comment
Username: @kairion
> When touching "Adicionar RG Digital" with AdGuard enabled I get the message "No internet connection". Right after disabling AdGuard I can proceed and add my digital ID without any issues.
> 
> The same happens when updating a digital ID that is already registered.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://reports-img.adguard.com/QeL3nWQ.png)
</details>
<details>
  <summary>Screenshot 2</summary>

  ![Screenshot 2](https://reports-img.adguard.com/pBh7FJr.png)
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 9
AdGuard product: | AdGuard for Android v3.6.4 (32)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
Stealth mode: | disabled
DNS filtering: | server: `System`<br>filters: `https://filters.adtidy.org/android/filters/15_optimized.txt`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>Fanboy's Annoyances<br/><br/><b>Security:</b><br/>NoCoin Filter List, <br/>Online Malicious URL Blocklist<br/><br/><b>Language-specific:</b><br/>AdGuard Spanish/Portuguese
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js

**Comment by Alex-302** [comment]
Cannot find APK to test, this app is not available for my device.

**Comment by Birbber** [comment]
@kairion Hi! Unfortunately we weren't able to install apk.
Try to disable filters one by one to see what filter might disturb the application. As a next step you can try disabling filtration for the application in Apps Management.
If that doesn't help try disabling DNS filtration. Let us know the result.

**Comment by Birbber** [comment]
@kairion Any news?

**Comment by Chinaski1** [comment]
In the end, I managed to find the [right application](https://uploads.adguard.com/Chinaskijdx09.pngIn), but I did not encounter the issues that you described.

To troubleshoot this issue, we need to get additional logs.

Here's what we need you to do:

1. Collect the debug log as it's [explained here](https://kb.adguard.com/en/android/solving-problems/log#debug);
2. Remember the exact time when the issue was reproduced. We will need it to find the corresponding records in the log file;
3. Additionally, capture the system bug report after reproducing the issue. [Here's the instruction](https://developer.android.com/studio/debug/bug-report);
4. Send both files to `devteam@adguard.com`. Mention the Github issue number and the exact time when the issue was reproduced.
